packages/calcite-components/src/components/navigation-user/navigation-user.e2e.ts

```diff
@@ -39,4 +41,40 @@ describe("calcite-navigation-user", () => {
   describe("is focusable", () => {
     focusable("calcite-navigation-user");
   });
+
+  describe("theme", () => {
+    const navigationUserHtml = html`
+      <calcite-navigation-user full-name="Walt McChipson" username="waltChip"> </calcite-navigation-user>
+    `;
+
+    describe("default", () => {
+      const tokens: ComponentTestTokens = {
+        "--calcite-navigation-user-avatar-corner-radius": {
+          shadowSelector: `calcite-avatar`,
+          targetProp: "--calcite-avatar-corner-radius",
+        },
+        "--calcite-navigation-user-avatar-text-color": {
+          shadowSelector: `calcite-avatar`,
+          targetProp: "--calcite-avatar-text-color",
+        },
+        "--calcite-navigation-user-background-color": {
+          shadowSelector: `.${CSS.button}`,
+          targetProp: "backgroundColor",
+        },
+        "--calcite-navigation-user-border-color": {
+          shadowSelector: `.${CSS.button}`,
+          targetProp: "borderBlockEndColor",
+        },
+        "--calcite-navigation-user-full-name-text-color": {
+          shadowSelector: `.${CSS.button} >>> .${CSS.fullName}`,
+          targetProp: "color",
+        },
+        "--calcite-navigation-user-name-text-color": {
```

Comment:
as with the `calcite-avatar` selector these should target the sub-component's tokens

Reply:
There isn't any sub-component other `calcite-avatar` in `navigation-user`. Updated this to use class selector without piercing selector.